### PR TITLE
Add "breaking semver by adding import or private type" post.

### PR DESCRIPTION
**Pull request by obi1kenobi**

### CODE REVIEW

File: draft/2023-05-10-this-week-in-rust.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ and just ask the editors to select the category.
 ### Project/Tooling Updates
 
 ### Observations/Thoughts
+- [Breaking semver in Rust by adding a private type, or by adding an import](https://predr.ag/blog/breaking-semver-in-rust-by-adding-private-type-or-import/)
```
Comment (mariannegoldin) [inline draft/2023-05-10-this-week-in-rust.md:39]: Tiny formatting change
```suggestion
* [Breaking semver in Rust by adding a private type, or by adding an import](https://predr.ag/blog/breaking-semver-in-rust-by-adding-private-type-or-import/)
```

**Review by mariannegoldin — APPROVED** [review]
Thought provoking and interesting post! Thank you for submitting. Approved with small formatting change.

**Review by mariannegoldin — APPROVED** [review]
Thought provoking and interesting post! Thank you for submitting. Approved with small formatting change.

**Comment by obi1kenobi** [comment]
Thank you for checking it out and for fixing the formatting ❤️

**Pull request merged**